vwr/driver.py

```diff
@@ -42,6 +49,16 @@ def _connect(self):
         self.waiting = False
         if self.timeout:
             self.listener.settimeout(self.timeout)
+        self.connected = True
+
+    def _reconnect(self):
+        """Reconnects on decay to the bath"""
+        try:
+            self._connect()
+            self.get_setpoint()
+        except socket.timeout:
+            self.delay = min(self.delay * self.factor, self.max_delay)
+            Timer(self.delay, self._reconnect).start()
```

Comment:
How will this tell the user if it's disconnected? It can return `{'connected': False}` when it can't read, but you need to do that somewhere here. And make sure it works for all functions.